webapp/src/store/transfer-courses/actions.ts

```diff
@@ -0,0 +1,13 @@
+import {createAction} from 'redux-api-middleware';
+import {ENDPOINT} from '../../lib/config';
+import {GET_TRANSFER_COURSES_REQUEST, GET_TRANSFER_COURSES_SUCCESS, GET_TRANSFER_COURSES_ERROR} from './types';
+
+export const getCourses = () => createAction({
+  endpoint: `${ENDPOINT}/api/courses/?limit=12000`,
```

Comment:
```suggestion
  endpoint: `${ENDPOINT}/api/courses/?limit=15000`,
```

Let's bump this up some more. Currently scraped classes returns over 12,000 courses.